compiler/data/function-data.h

```diff
@@ -145,8 +150,9 @@ class FunctionData {
 
   string get_resumable_path() const;
   string get_throws_call_chain() const;
-  static string get_human_readable_name(const std::string &name);
-  string get_human_readable_name() const;
+  std::string get_performance_inspections_warning_chain(PerformanceInspections::Inspections inspection, bool search_disabled_inspection = false) const noexcept;
+  static string get_human_readable_name(const std::string &name, bool add_details = true);
```

Comment:
btw, a static function seems to be useless, as it's called only from an instance one

Reply:
unfortunately it is used in preprocess-function.cpp

Reply:
😳 CLion didn't show me that usage, okay..